### ISSUE TITLE
ssl conformer learning rate & warmup

### ISSUE THREAD

**Issue by alexandercesarr** [NVIDIA/NeMo#5519]
Hi, I've trained a **SSL conformer** model. Now I wanna **fine-tune** it with labeled data.
I encountered two problems:

1. I want the learning rate decrease linearly.
I don't want warmup. So I set it 0.
But in this situation, it first increase and then decrease.

2. the second problem is I want to control the learning rate decrease speed. For this one I changed the weight_decay and also min_lr but it decreased very fast. How can I control it?
I want it be constant.

**Comment by titu1994** [comment]
Sorry for the delay. It is possible that you are not updating the value of model.optim using change_optimization() @sam1373
